core/CronArchive.php

```diff
@@ -715,6 +715,12 @@ public function logError($m)
 
     private function logNetworkError($url, $response)
     {
+
+        if (preg_match("/Segment (.*?) is not a supported segment/i", $response, $match)) {
+            $this->logger->info($match[0]);
```

Comment:
That already looks fine. But the log then contains 
```
Segment 'languageCode' is not a supported segment\n
Archived website id 1, period = day, date = 2019-12-12, segment = 'languageCode==fr', 0 visits found. Time elapsed: 4.590s\n
```

Might be fine, but actually it didn't archive anything as the segment was invalid.
If easily possible it would be nice to have something like this instead:

```
Segment 'languageCode' is not a supported segment\n
Skipped archiving website id 1, period = day, date = 2019-12-12, segment = 'languageCode==fr'. Time elapsed: 4.590s\n
```